### PR TITLE
1936 feature add input recipe

### LINKED ISSUE #1936: [Feature?]: add input recipe
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Summary 💡

- [x] Recipe only
- [x] Sandbox usage
- [ ] Sandbox tests

### Examples 🌈

_No response_

### Motivation 🔦

Preset, sandbox

### PR DESCRIPTION
**Pull request by caseybaggz**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
closes #1936 

## What is the new behavior?
- Adds input recipe
- Adds new `_userInvalid` condition
- Updates sandbox
- Adds initial sandbox tests

### CODE REVIEW

File: sandbox/app/react/input/page.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    return pandoSize === 'md' ? input({ size: 'md' }) : input()
+  }, [pandoSize])
+  const iconSize = useMemo(
+    () => (pandoSize === 'md' ? '1rem' : '1.5rem'),
+    [pandoSize],
+  )
+
+  return (
+    <div className={styles.root}>
+      <input className={cx('peer', styles.control)} {...nativeProps} />
```
[the remark this answers is not in the document]
Comment (caseybaggz) [reply sandbox/app/react/input/page.tsx:45]: This is temporary

File: sandbox/app/react/input/page.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+export default function ButtonPage() {
+  const [value, setValue] = useState('')
+
+  function handleChange(e: ChangeEvent<HTMLInputElement>) {
+    setValue(e.currentTarget.value)
+  }
+
+  return (
+    <>
```
Comment (kathydlavoie-ps) [inline sandbox/app/react/input/page.tsx:64]: jw - why wrap the section in a fragment? the section is a single wrapper which should satisfy the single exported component requirement - is there a specific reason in this case?

**Review by kathydlavoie-ps — APPROVED** [review]
non blocking q!

File: sandbox/app/react/input/page.tsx (hunk 2 above)
Comment (caseybaggz) [reply sandbox/app/react/input/page.tsx:64]: Because in my next PR I'll be adding multiple sections. 😉

**Pull request merged**